client/client.go

```diff
@@ -85,9 +87,35 @@ func NewClient(cfg *configuration.Configuration, rpcClient *RPCClient) (*SDKClie
 		RPCClient:      c,
 		EthClient:      ec,
 		traceSemaphore: semaphore.NewWeighted(maxTraceConcurrency),
+		maxBatchSize:   cfg.MaxBatchSize,
 	}, nil
 }
 
+// MaxBatchSize returns the maximum number of requests to batch together.
+func (ec *SDKClient) MaxBatchSize() int {
+	return ec.maxBatchSize
+}
+
+// BatchCallContext executes a batch of RPC calls.
+// Overrides the [RPCClient] BatchCallContext method to limit the number of batched requests.
```

Comment:
Do you know what is the default in `[RPCClient] BatchCallContext`? or it does not have a limit?